src/transformers/models/qwen2_5_omni/processing_qwen2_5_omni.py

```diff
@@ -174,8 +176,7 @@ def __call__(
         if videos is not None:
             videos = make_batched_videos(videos)
             videos_inputs = self.image_processor(images=None, videos=videos, **output_kwargs["videos_kwargs"])
-            if fps is None:
-                fps = [2.0] * len(videos)
+            fps = [fps] * len(videos)
```

Comment:
This is technically unrelated, but I don't think the input kwarg should be expected as a list here.